eclass/git-r3.eclass

```diff
@@ -905,6 +972,13 @@ git-r3_checkout() {
 		fi
 		echo "${@}" >&2
 		"${@}" || die "git checkout ${remote_ref:-${new_commit_id}} failed"
+
+		# If any filters in any .gitattributes file specifies lfs, this repo is most likely storing files with git lfs.
+		local has_git_lfs_filters=$(cd "${out_dir}" && git ls-files .gitattributes '**/.gitattributes' | xargs grep filter=lfs)
```

Comment:
Please use line wrapping.
```suggestion
		local has_git_lfs_filters=$(
			cd "${out_dir}" &&
			git ls-files .gitattributes '**/.gitattributes' |
				xargs grep filter=lfs
		)
```

Reply:
Also, `git grep` perhaps?

Reply:
Gah! I had a feeling I had forgotten a better command for this.
Thanks for pointing this out. Now the code it a lot less complex! :)